spec/unit/check_board_spec.rb

```diff
@@ -30,9 +44,195 @@
+                                Player.new(:X, 1, 2),
+                                Player.new(:O, 1, 0),
+                                Player.new(:X, 2, 2)
+                              ])
+      view_board = ViewBoard.new(player_gateway: player_gateway)
+      response = view_board.execute
+      board = response[:board]
+      expect(check_board.execute(board)).to eq('X has won')
+    end
+
+    it 'can return X as the winner from horizontal top ' do
+      player_gateway = double(placing_xos: [
+                                Player.new(:X, 0, 0),
+                                Player.new(:O, 1, 1),
+                                Player.new(:X, 0, 1),
+                                Player.new(:O, 1, 0),
+                                Player.new(:X, 0, 2)
+                              ])
+      view_board = ViewBoard.new(player_gateway: player_gateway)
+      response = view_board.execute
+      board = response[:board]
+      expect(check_board.execute(board)).to eq('X has won')
+    end
+
+    it 'can return X as the winner from horizontal middle ' do
+      player_gateway = double(placing_xos: [
+                                Player.new(:X, 1, 0),
+                                Player.new(:O, 0, 1),
+                                Player.new(:X, 1, 1),
+                                Player.new(:O, 2, 0),
+                                Player.new(:X, 1, 2)
+                              ])
+      view_board = ViewBoard.new(player_gateway: player_gateway)
+      response = view_board.execute
+      board = response[:board]
+      expect(check_board.execute(board)).to eq('X has won')
+    end
+
+    it 'can return X as the winner from horizontal bottom ' do
+      player_gateway = double(placing_xos: [
+                                Player.new(:X, 2, 0),
+                                Player.new(:O, 1, 1),
+                                Player.new(:X, 2, 1),
+                                Player.new(:O, 1, 0),
+                                Player.new(:X, 2, 2)
+                              ])
+      view_board = ViewBoard.new(player_gateway: player_gateway)
+      response = view_board.execute
+      board = response[:board]
+      expect(check_board.execute(board)).to eq('X has won')
+    end
+  end
+
+  context 'Winning for O' do
+    it 'can return O as the winner from vertical left ' do
+      player_gateway = double(placing_xos: [
+                                Player.new(:O, 0, 0),
+                                Player.new(:X, 0, 1),
+                                Player.new(:O, 1, 0),
+                                Player.new(:X, 1, 2),
+                                Player.new(:O, 2, 0)
+                              ])
+      view_board = ViewBoard.new(player_gateway: player_gateway)
+      response = view_board.execute
+      board = response[:board]
+      expect(check_board.execute(board)).to eq('O has won')
+    end
+
+    it 'can return O as the winner from vertical middle' do
```

Comment:
these tests can probably be refactored to reduce duplication.

Your it blocks can be simplified

e.g. 

```ruby
expect_check_board_to_respond_with(
  :o_wins, 
  [
    Player.new(:O, 0, 1),
    Player.new(:X, 0, 0),
    Player.new(:O, 1, 1),
    Player.new(:X, 1, 2),
    Player.new(:O, 2, 1)
  ]
)
```